packages/vuetify/src/directives/color/__tests__/color.spec.ts

```diff
@@ -0,0 +1,141 @@
+// Vue
+import Vue from 'vue'
+
+// Directives
+import Color from '../'
+
+// Utilities
+import {
+  mount,
+  Wrapper,
+} from '@vue/test-utils'
+
+interface BorderModifiers {
```

Comment:
can we get rid of the duplication? this interface is already defined in the directive file